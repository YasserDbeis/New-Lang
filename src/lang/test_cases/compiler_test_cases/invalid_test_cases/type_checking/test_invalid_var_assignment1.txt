func main()
{
    int x = 10.0;
}