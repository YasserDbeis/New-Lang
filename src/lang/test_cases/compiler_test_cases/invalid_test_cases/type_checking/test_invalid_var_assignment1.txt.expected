RUNTIME EXCEPTION
Error occurred in Execution
At expression 10.0
Expression has invalid evaluation
3